issue/views.py

```diff
@@ -0,0 +1,89 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth.models import User
+from django.contrib.auth.decorators import login_required, permission_required
+from django.core.paginator import Paginator
+from django.db.models import Count
+from .models import *
+from .forms import *
```

Comment:
ここは必要なモジュールのみimportしたほうが良いと思います。
使う時に何をimportしたか分からなくなるので。
```suggestion
from .models import Status, Issue
from .forms import IssueForm, IssueEditForm
```